js/test/date/testWeekdayTranslation.js

```diff
@@ -11151,6 +11151,62 @@ module.exports.testWeekdayTranslation = {
         test.equal(value[5], "五");
         test.equal(value[6], "六");
         
+        test.done();
+    },
+    testWeekdayTranslation_ka_GE: function(test) {
+        // full -> wide, long -> abbreviate
+        // medium -> short
+        // short: narrow
+        test.expect(28);
+        var fmt, value = [], i;
+        fmt = new DateFmt({locale:"ka-GE", date:"w", length: "full", useNative:false, timezone:"local"})
+        for (i=0; i < 7; i++) {
+            value[i] = fmt.format(DateFactory({year: 2015, month: 8, day:i+2, type:"gregorian"}));
+        }
+        test.equal(value[0], "კვირა");
+        test.equal(value[1], "ორშაბათი");
+        test.equal(value[2], "სამშაბათი");
+        test.equal(value[3], "ოთხშაბათი");
+        test.equal(value[4], "ხუთშაბათი");
+        test.equal(value[5], "პარასკევი");
+        test.equal(value[6], "შაბათი");
+
+        fmt = new DateFmt({locale:"ka-GE", date:"w", length: "long", useNative:false, timezone:"local"})
```

Comment:
This is testing 4 different things. There should be 4 unit tests: Full, Long, Medium, and Short

Reply:
Regarding the format,
```
full, long --> MMMM
medium -->  MMM
short -->  MM
```
full and long are the same, and short will be written the number. That's why only two length test cases are written.

Reply:
What I meant is to break that 1 big unit tests into 4 separate ones, so if any of them fail, then it is easier to figure out what went wrong.

Reply:
oh.. sorry, ;(  I should've read carefully and checked the correct file. following your comment, I re-updated it. Thank you